ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/ErrorSection.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+    Bullseye,
+    EmptyState,
+    EmptyStateVariant,
+    EmptyStateIcon,
+    Title,
+    EmptyStateBody,
+} from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+
+export type ErrorSectionProps = {
+    error: Error;
+    children?: React.ReactNode;
+};
+
+function ErrorSection({ error, children }: ErrorSectionProps) {
+    return (
+        <Bullseye>
+            <EmptyState variant={EmptyStateVariant.large}>
+                <EmptyStateIcon className="pf-u-danger-color-100" icon={ExclamationCircleIcon} />
+                <Title headingLevel="h2">{getAxiosErrorMessage(error)}</Title>
+                {children && <EmptyStateBody>{children}</EmptyStateBody>}
+            </EmptyState>
+        </Bullseye>
+    );
+}
+
+export default ErrorSection;
```

Comment:
oh wow i'm surprised we don't have a version of this already. might be worth pulling it out to the `Components` directory for broader use

Reply:
Well there is the `EmptyStateTemplate`, which is close but not quite compatible. I can see what the change would look like if I enabled that component to accept a `iconClassName` prop and wrapped it in a `Bullseye` at each call site.